JitsiConference.js

```diff
@@ -1439,15 +1429,27 @@ JitsiConference.prototype.getPhonePin = function() {
 };
 
 /**
+ * Obtains currently active <tt>TraceablePeerConnection</tt> (if any).
+ * @return {TraceablePeerConnection|null}
+ * @protected
+ */
+JitsiConference.prototype.getActivePeerConnection = function() {
+    return this.jvbJingleSession
+        ? this.jvbJingleSession.peerconnection : null;
```

Comment:
will it return null ? do I understand correctly that with && it will return this.jvb.JingleSession ? So if it's null it should return null ?